### ISSUE TITLE
Look into Android Studio dependencies for dev collab 

### ISSUE THREAD

**Issue by NerdboyQ** [NIAGroup/Team_App_Android#3]
Adonay has brought up a concern about potential collaboration issues with app dependencies. We should consider investigating this before collaboration efforts begin (github push/pull/commits).

**Comment by NerdboyQ** [comment]
@adonaygyb I was able to clone and run the hello world app with no problems. I'll wait for the other team members to try it out before closing.

**Comment by adonaygyb** [comment]
@Injoh @jmpichar @kaurnav1227 are we good to go here?

**Comment by adonaygyb** [comment]
@NerdboyQ, i don't believe this is an issue anymore. I looked a bit into this and it seems like source control (Git) should be all we need for a multi-person development project in Android Studio. 
https://www.quora.com/How-can-multiple-developers-collaborate-to-develop-an-Android-app

Furthermore, i found this cool plugin that enables real-time collaboration in the same IDE that should be available in Android Studio (i haven't played with it yet). I don't think we need it yet, but it's nice to know as an option. 
https://floobits.com/help/plugins/intellij

**Comment by adonaygyb** [comment]
I move to closing this issue if everyone i sin agreement.
